src/System.Private.CoreLib/shared/System/Globalization/CultureData.Windows.cs

```diff
@@ -55,7 +56,7 @@ private unsafe bool InitCultureData()
             Debug.Assert(!GlobalizationMode.Invariant);
 
             int result;
-            string realNameBuffer = _sRealName;
+            string realNameBuffer = _sRealName!;
```

Comment:
why is _sRealName nullable?

Reply:
Do we instead need to initialize `_sRealName` with `null!`?

Reply:
actually good point, shouldn't be nullable but instead initialized with `null!`